clients/unshaded/src/main/java/tachyon/client/netty/NettyRemoteBlockReader.java

```diff
@@ -38,11 +39,13 @@
 /**
  * Read data from remote data server using Netty.
  */
-public final class NettyRemoteBlockReader implements RemoteBlockReader {
+public final class NettyRemoteBlockReader implements Closeable, RemoteBlockReader {
```

Comment:
I'm fine either way, the other readers (i.e. TCPRemoteBlockReader) are currently stateless. I have made `RemoteBlockReader` to extend `Closeable` so that we don't need to check `instanceof NettyRemoteBlockReader`.